wiki/Rules/en.md

```diff
@@ -1,77 +1,79 @@
-6.  Signatures should not exceed 256kb in total filesize. Animations are permitted, so long as they are reasonable and don't contain photosensitive elements (quick flashing or strobing).
-7.  Animated avatars are tolerated, but have the same regulation as signatures. Avatars must always be suitable for all ages.
-8.  Advertisements and referral links are explicitly forbidden. Just don't post them, period.
-9.  Autoloading content (youtube videos, flash applets, etc) is not allowed.
-
-BSS Forum Rules
+1.  **Post only meaningful, interesting or otherwise engaging content/topics.** Forums are about discussions foremost, so make sure to put the effort in to make it worth everyone's time.
+2.  **Keep your posts and replies suitable for people of all-ages.** NSFW content isn't appropriate for an all-ages community, though there's nothing stopping you from discussing these sorts of things in private messages.
+3.  **Be productive with your criticism without resorting to personal attacks.** Criticism is a wonderful thing when done properly, but if you're resorting to personal attacks to make your point, you're doing it wrong, and you should feel bad.
+4.  **Keep threads relevant, and avoid resurrecting older topics if the conversation has died out.** If a topic has been dead for years, it isn't really relevant anymore, so just let it fade away. Post a new one, or maybe search and see if someone has answered your question already.
+5.  **Keep your signatures and avatars reasonable.** If they're animated, they shouldn't strobe or flash excessively, and they should be of a filesize small enough for users with slower connections to not feel their presence on the page.
+6.  **Don't post advertisements, referral links or any other annoying or intrusive content.** Things like affiliate and referral links are fun for the person posting them and an annoyance for everybody else, so don't do it.  Autoloading or autoplaying content/embeds falls within this rule as well.
+7.  **Linking to applications or utilities of uncertain intent is not allowed.** Put simply, if there is any doubt as to the legality of said utilities, they are not welcome to be posted.
+
+[Beatmap Submission Rules](BSS "wikilink")
 ---------------------------------
 
-These rules apply strictly to the unique nature of the [BSS](/wiki/Glossary) ([Beatmap](/wiki/Glossary) Submission System) threads. These threads are created via the osu! client when a beatmap is submitted through the [editor](/wiki/Beatmap_Editor).
+These rules apply strictly to beatmaps submitted via the in-game [Beatmap Submission System](BSS "wikilink"). These threads/discussions are created via the osu! client when a beatmap is submitted through the [editor](editor "wikilink").
+
+A set of more detailed rules for beatmap construction exists, known as the [Ranking Criteria](Ranking_Criteria "wikilink").
```

Comment:
Fix this link:
- `[Ranking Criteria](Ranking_Criteria "wikilink")` ↙️
  - `[Ranking Criteria](/wiki/Ranking_Criteria)`